### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by ash3rz**
I have another branch with a few more warning fixes, but I'd rather save those for after the release just in case

### CODE REVIEW

File: react-components/stories/analysis/view/dialogs/ViceLogsViewerDialog.stories.js [hunk 1]
```diff
@@ -10,6 +10,19 @@ class ViceLogsViewerTest extends Component {
         window.setTimeout(() => {
             this.setState({ mask: false });
         }, 3000);
+
+        const presenter = {
```
Comment (psarando) [inline react-components/stories/analysis/view/dialogs/ViceLogsViewerDialog.stories.js]: 👍

File: react-components/stories/apps/quickLaunch/CreateQuickLaunchDialog.stories.js [hunk 2]
```diff
@@ -3,11 +3,20 @@ import CreateQuickLaunchDialog from "../../../src/apps/quickLaunch/CreateQuickLa
 
 class CreateQuickLaunchDialogTest extends Component {
     render() {
+        const presenter = {
+            createQuickLaunch: (name, description, isPublic) => {
+                console.log("Create Quick Launch", name, description, isPublic);
```
Comment (psarando) [inline react-components/stories/apps/quickLaunch/CreateQuickLaunchDialog.stories.js]: 👍

**Comment by sri2k1us** [comment]
LGTM :shipit:

**Comment by ash3rz** [comment]
Thanks for the reviews!  I'll rebase on top of master since this is behind now, and then I'll bump up the ui-lib version as well before merging.

**Pull request merged**